easybuild/easyblocks/generic/binariestarball.py

```diff
@@ -40,21 +41,43 @@ class BinariesTarball(Tarball):
     """
     Support for installing a tarball of binaries
     """
+    @staticmethod
+    def extra_options(extra_vars=None):
+        """
+        Define list of files to be copied during the installation process
+        """
+        extra_vars = Tarball.extra_options(extra_vars)
+        # Allow specification of files to copy explicitly.
+        # If files_to_copy is a zero-length list, all files (but not
+        # directories, etc.) in start_dir will be copied with their
+        # current names.
+        extra_vars['files_to_copy'] = [
+            [],
```

Comment:
Precedents elsewhere seem to be in favour of an empty list (see e.g. the `makesymlinks` extra variable in `rpm.py`).